userspace/libsinsp/test/savefile.ut.cpp

```diff
@@ -43,20 +44,26 @@ TEST(savefile, filter)
 		sinsp inspector;
 		inspector.set_filter("proc.name=ifplugd");
 		inspector.open_savefile(RESOURCE_DIR "/sample.scap");
-		inspector.setup_cycle_writer(filtered_scap, 0, 0, 0, 0, true);
-		inspector.autodump_next_file();
+
+		sinsp_cycledumper* dumper = new sinsp_cycledumper(&inspector, filtered_scap,
```

Comment:
Also use smart pointer in the test, but actually can we make the test more realistic and call `sinsp_cycledumper::autodump_start` instead etc?

Reply:
Double-checking re making the test more realistic? Any updates on that?

Reply:
Didn’t forget about that! I’m looking into it! :)